packages/arb-util/configuration/configuration.go

```diff
@@ -702,6 +721,18 @@ func ParseNonRelay(ctx context.Context, f *flag.FlagSet, defaultWalletPathname s
 	if out.Node.Cache.AllowSlowLookup {
 		// Force unlimited execution
 		out.Core.CheckpointMaxExecutionGas = 0
+
+		// Never prune checkpoints
+		out.Core.CheckpointPruningMode = "off"
```

Comment:
If we're overriding the current setting, maybe good to print a log line?

Reply:
:+1: